### PR TITLE
Using stamp - docs

### PR DESCRIPTION
**Pull request by newmedicine**
My first attempt at improving the docs. Thoughts welcome. Happy New Year!

### CODE REVIEW

File: editions/tw5.com/tiddlers/howtos/Using Stamp.tid [hunk 1]
```diff
@@ -1,27 +1,59 @@
...
+## Enter `caption` in the //Add new field// `field name` box
+## Enter the stamp menu item caption in the `field value` box
+
+
+
+! Re-ordering snippets
+Your new snippet will appear at the bottom of the stamp menu by default. To adjust the order of snippets:
+
+# Navigate to a snippet tiddler
+# Click the <<tag-pill $:/tags/TextEditor/Snippet>> tag pill
```
Comment (pmario) [inline editions/tw5.com/tiddlers/howtos/Using Stamp.tid:40]: hmm, It describes the "re-ordering" using the `<<tag` macro, but it doesn't work here. IMO the `<<tag` macro should be used. So reordering works here too.

File: editions/tw5.com/tiddlers/howtos/Using Stamp.tid [hunk 2]
```diff
@@ -1,27 +1,59 @@
...
+# Drag items up and down the menu of snippets
+
+
+! <span style="font-size:1rem;"><<.from-version "5.1.20">></span>&nbsp;Adding a prefix and/or suffix to a selection
```
Comment (pmario) [inline editions/tw5.com/tiddlers/howtos/Using Stamp.tid:44]: IMO this looks strange. May be the `<<.from-version` should be below the heading as a single new line. It should make it clear the the whole "section" is/was new with 5.1.20

image

Comment (pmario) [reply editions/tw5.com/tiddlers/howtos/Using Stamp.tid:44]: May be the "new in" info could be part of the heading, but //at the end//

**Review by newmedicine — COMMENTED** [review]
OK I've made the two changes suggested, thanks

File: editions/tw5.com/tiddlers/howtos/Using Stamp.tid (hunk 2 above)
Comment (newmedicine) [reply editions/tw5.com/tiddlers/howtos/Using Stamp.tid]: OK I think I've made the corrections ow

**Comment by pmario** [comment]
Looks good to me. Happy new year :)

**Comment by Jermolene** [comment]
Thank you @newmedicine @pmario these are very worthwhile improvements, much appreciated.

**Pull request merged**